### ISSUE TITLE
Required roles for aria-controls

### ISSUE THREAD

**Issue by geoffrey-eisenbarth** [w3c/aria#2724]
# Describe your concern
Currently `aria-controls` is considered a global attribute, which seems odd given that the nature of the property implies interactivity. 

A list of allowable roles for `aria-controls` was once proposed (https://github.com/w3c/aria/issues/999) and a later issue (https://github.com/w3c/aria/issues/1089) suggested that going forward, individual issues should be made... so this is my attempt to do just that!

# Link to the version of the specification or documentation you were looking at at.

Link to documentation: https://w3c.github.io/aria/#aria-controls

Does the issue exists in [the editors draft](https://w3c.github.io/aria/) (the editors draft is the most recent draft of the specification)? **Yes**

**Comment by pkra** [comment]
Context from https://github.com/w3c/aria/issues/999#issue-458807533

> Note, https://github.com/w3c/aria/issues/995 may mean aria-controls should remain global until a larger discussion on if any redefinition of the attribute should take place so that AT will be more inclined implement it / not ignore it.

The main reason was lack in AT support. 

#995 was closed via https://github.com/w3c/aria/pull/1996 last year, so there has been a significant update.

**Comment by aardrian** [comment]
I think this issue is dependent on UAs and AT indicating what support they will add.
